JS/app.js

```diff
@@ -0,0 +1,49 @@
+let searchButton = document.getElementById("searchButton").addEventListener('click', Search);
+let loadingSpot = document.getElementById("loading")
+let loadingIndicator = `<div class="d-flex justify-content-center"><div class="spinner-grow text-info" role="status">
+<span class="sr-only">Loading...</span>
+</div></div>`;
+
+function Search(e) {
```

Comment:
don't start function name with capital letters. this is reserved in JS to object constractors.

Reply:
Corrected.